face-detection-explainer.md

```diff
@@ -0,0 +1,481 @@
+# Face Detection
+
+## Authors:
+
+- Rijubrata Bhaumik, Intel Corporation
+- Tuukka Toivonen, Intel Corporation
+- Eero Häkkinen, Intel Corporation
+
+## Participate
+- github.com/riju/faceDetection/issues/
+
+## Introduction

```

Comment:
Should have a "status of this memo" section saying that this is part of a proposal to the WG, not an item / approach accepted by the WG.

Reply:
Resolved by added commit